sdk/storage/storage-queue/review/generated/models/index.ts

```diff
@@ -0,0 +1,1450 @@
+   */
+  timeoutParameter?: number;
+  /**
+   * Optional. Include this parameter to specify that the queue's metadata be returned as part of
+   * the response body. Note that metadata requested with this parameter must be stored in
+   * accordance with the naming restrictions imposed by the 2009-09-19 version of the Queue
+   * service. Beginning with this version, all metadata names must adhere to the naming conventions
+   * for C# identifiers.
+   */
+  metadata?: { [propertyName: string]: string };
+  /**
+   * Provides a client-generated, opaque value with a 1 KB character limit that is recorded in the
+   * analytics logs when storage analytics logging is enabled.
+   */
+  requestId?: string;
+}
+
+/**
+ * Optional Parameters.
+ */
+export interface QueueGetAccessPolicyOptionalParams extends coreHttp.RequestOptionsBase {
+  /**
+   * The The timeout parameter is expressed in seconds. For more information, see <a
+   * href="https://docs.microsoft.com/en-us/rest/api/storageservices/setting-timeouts-for-queue-service-operations>Setting
+   * Timeouts for Queue Service Operations.</a>
+   */
+  timeoutParameter?: number;
+  /**
+   * Provides a client-generated, opaque value with a 1 KB character limit that is recorded in the
+   * analytics logs when storage analytics logging is enabled.
+   */
+  requestId?: string;
+}
+
+/**
+ * Optional Parameters.
+ */
+export interface QueueSetAccessPolicyOptionalParams extends coreHttp.RequestOptionsBase {
+  /**
+   * the acls for the queue
+   */
+  queueAcl?: SignedIdentifier[];
+  /**
+   * The The timeout parameter is expressed in seconds. For more information, see <a
+   * href="https://docs.microsoft.com/en-us/rest/api/storageservices/setting-timeouts-for-queue-service-operations>Setting
+   * Timeouts for Queue Service Operations.</a>
+   */
+  timeoutParameter?: number;
+  /**
+   * Provides a client-generated, opaque value with a 1 KB character limit that is recorded in the
+   * analytics logs when storage analytics logging is enabled.
+   */
+  requestId?: string;
+}
+
+/**
+ * Optional Parameters.
+ */
+export interface MessagesDequeueOptionalParams extends coreHttp.RequestOptionsBase {
+  /**
+   * Optional. A nonzero integer value that specifies the number of messages to retrieve from the
+   * queue, up to a maximum of 32. If fewer are visible, the visible messages are returned. By
+   * default, a single message is retrieved from the queue with this operation.
+   */
+  numberOfMessages?: number;
+  /**
+   * Optional. Specifies the new visibility timeout value, in seconds, relative to server time. The
+   * default value is 30 seconds. A specified value must be larger than or equal to 1 second, and
+   * cannot be larger than 7 days, or larger than 2 hours on REST protocol versions prior to
+   * version 2011-08-18. The visibility timeout of a message can be set to a value later than the
+   * expiry time.
+   */
+  visibilitytimeout?: number;
```

Comment:
I think this should be visibility**T**imeout

Reply:
#5553